responses.json

```diff
@@ -0,0 +1,19 @@
+{
+  "lit": {
```

Comment:
Good idea adding `success` and `failure` under `lit`. This way, we can get subclass these strings.